src/test/java/com/microsoft/azure/spring/data/documentdb/TestUtils.java

```diff
@@ -17,4 +22,25 @@
         }
         return null;
     }
+
+    public static <T extends JsonSerializable> void testIndexingPolicyPaths(Collection<T> policyPaths,
```

Comment:
change `testIndexingPolicyPaths` to `testIndexingPolicyPathsEquals`?

Reply:
copy that.